integration/simple-cache-client.test.ts

```diff
@@ -60,87 +67,151 @@ describe('SimpleCacheClient.ts Integration Tests', () => {
     await withCache(momento, cacheName, async () => {
       await momento.set(cacheName, 'key', 'value');
       const res = await momento.get(cacheName, 'key');
-      expect(res.text()).toEqual('value');
+      expect(res instanceof CacheGet.Error).toEqual(false);
+      if (res instanceof CacheGet.Hit) {
+        expect(res.valueString()).toEqual('value');
+      }
```

Comment:
If it isn't a CacheGet.Hit, the test will pass. Same for the rest of the tests.

Add an expectation.

```suggestion
      expect(res instanceof CacheGet.Hit).toEqual(true)
      expect(res.valueString()).toEqual('value');
```

Reply:
I made a similar comment farther down, but I do not believe this would compile because the parent class does not have a method called `valueString`.

Reply:
The line before this `if` asserts that the response is not an `.Error` type, so it will fail if an error is encountered. However, since this is a `get` test, it will give a false positive if the result is a miss, so I'll update for that.